conf.py

```diff
@@ -25,24 +30,19 @@
 }
 
 # List of patterns (relative to source dir) to ignore when looking for source files.
+include_patterns = [
+    # Required for Sphinx
+    "contents.rst",
+    # PEP files
+    "pep-????.???",
```

Comment:
```suggestion
    "pep-[0-9][0-9][0-9][0-9].rst",
    "pep-[0-9][0-9][0-9][0-9].txt",
```

More precise; right now it will pick up anything starting with `pep-` and four characters, with any standard three-character extension. E.g. `pep-0001.bak`, `pep-data.pkl`, `pep-0012.tmp`, `pep-help.rst`, etc. This just detects PEPs that match the format in PEP 1/ PEP 12.